src/DynamoCore/Graph/Nodes/ZeroTouch/DSFunction.cs

```diff
@@ -49,6 +51,28 @@ public DSFunction(FunctionDescriptor functionDescription)
             : base(new ZeroTouchNodeController<FunctionDescriptor>(functionDescription)) 
         {
         }
+
+        [NodeMigration(version: "2.0.0.0")]
+        public static NodeMigrationData MigrateGetImportedObjects(NodeMigrationData data)
+        {
+            var migrationData = new NodeMigrationData(data.Document);
+            var node = data.MigratedNodes.ElementAt(0);
+
+            if (node.GetAttribute("function").Equals("Dynamo.Translation.FileLoader.GetImportedObjects"))
```

Comment:
[This PR](https://git.autodesk.com/Dynamo/DynamoPro/pull/565) updates `Translation.Migrations.xml`.